test/test1.graphql

```diff
@@ -1,12 +0,0 @@
-query q1
```

Comment:
Was this file supposed to be removed?

Reply:
Well, was it @Bellk17 ? :)

Reply:
Original author had written a test query, but never implemented any tests.